aggregator/forms.py

```diff
@@ -47,3 +49,85 @@ def clean_feed_url(self):
                 "here automatically."
             )
         return feed_url
+
+
+class LocalDjangoCommunityModelForm(forms.ModelForm):
+    name = forms.CharField(
+        max_length=250,
+        widget=forms.TextInput(
+            attrs={
+                "class": "required",
+                "placeholder": "Name of the community",
+            }
+        ),
+    )
+    description = forms.CharField(
+        widget=forms.Textarea(
+            attrs={
+                "class": "required",
+                "placeholder": "Description of the community",
+            }
+        ),
+    )
+    continent = forms.CharField(
+        max_length=250,
+        widget=forms.TextInput(
```

Comment:
This should be `ChoiceField` similar to country.
Also I noticed that errors could have red color , because it was hard for me to notice them
![Zrzut ekranu z 2024-06-17 19-00-43](https://github.com/Alexander3/djangoproject.com/assets/4659834/eace426b-8616-48e3-baa6-961a66c7c545)